config.go

```diff
@@ -157,6 +161,9 @@ func parseDSNParams(cfg *Config, params map[string][]string) (err error) {
 			cfg.Params[k] = v[0]
 		case "tls_config":
 			cfg.TLSConfig = v[0]
+		case "priority":
+			cfg.Priority, err = strconv.Atoi(v[0])
+			cfg.Params[k] = v[0]
```

Comment:
Why does it  need in params?

Reply:
Yes, you are right we can put a the priority arg in Params. 
This PR is useless.
Thank you